src/ducks/components/intents/InstallAppIntent.jsx

```diff
@@ -29,7 +31,13 @@ export class InstallAppIntent extends Component {
   }
 
   componentDidMount() {
-    this.props.initAppIntent()
+    const { data } = this.props
+    const { activateDescriptionPageForSlug, category } = data || {}
```

Comment:
I think `activateDescriptionPageForSlug` is not a good API name :(. 

If in some scenario we need to go to an other page, we'll have to create an other props. Or will have to rename this one and then update other apps. Let's do that differently. Something like : pageToDisplay : ..., or 'routeToDisplay'

Reply:
C'est corrigé, nous utiliserons donc `pageToDisplay` 👍